go.mod

```
@@ -352,7 +352,7 @@ require (
 	gopkg.in/cheggaaa/pb.v1 v1.0.28 // indirect
 	gopkg.in/yaml.v2 v2.4.0 // indirect
 	gopkg.in/yaml.v3 v3.0.1 // indirect
-	howett.net/plist v0.0.0-20181124034731-591f970eefbb // indirect
+	howett.net/plist v0.0.0-20181124034731-591f970eefbb // indirect; indirect dependency-check-ignore: required by github.com/elastic/go-sysinfo
```

```suggestion
	howett.net/plist v0.0.0-20181124034731-591f970eefbb // indirect; dependency-check-ignore: required by github.com/elastic/go-sysinfo
```

extra `indirect` is redundant?

---

yeah, that can be fixed, but this PR is full of conflicts now the base has shifted so I'm going to leave it to @galargh to resolve if he pulls this in, otherwise he can write it the way he wants